ci/build-common.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e
```

Comment:
this suppose to be sources, so no need for shebang. Also for the record, I find it more complex for unifying 4 common defines, but ok.